mycroft/dialog/__init__.py

```diff
@@ -108,3 +108,28 @@ def load(self, dialog_dir):
                 dialog_entry_name, os.path.join(dialog_dir, f))
 
         return self.__renderer
+
+
+def get(phrase, lang, context={}):
+    """
+    Looks up a resource file for the given phrase.  If no file
+    is found, the requested phrase is returned as the string.
+    This will use the default language for translations.
+
+    Args:
+        phrase (str): resource phrase to retrieve/translate
+        lang (str): the language to use
+        context (dict): values to be inserted into the string
+
+    Returns:
+        str: a randomized and/or translated version of the phrase
+    """
+
+    filename = resolve_resource_file("text/"+lang+"/"+phrase+".dialog")
+    print (filename)
```

Comment:
This is a debug message and I don't think it should be included. at least not as a *print*